CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ## [NEXT RELEASE]
 
 ### Added Features
+- ROX-18689: ACS will qualify the registry (and path) of images from the container runtime when env var `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is set to `true` on both Central and Sensor.
+  - This enables support for CRI-O's unqualified search registries and short name aliases ([more info](https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md)).
```

Comment:
The `imageID` field will exist regardless of the runtime, that's part of the K8s/OCP API.  I'm referring to the two features directly here since this was created to support CRI-O.  Believe ContainerD uses 'mirrors' to do similar - unknown if those will appear in the `imageID` or not.

Technically this would work for any runtime that populates `imageID`.